app/controllers/RouteController.scala

```diff
@@ -163,4 +163,21 @@ class RouteController @Inject()(routeService: RouteService,
     }
   })
 
+  def getRoutes() = securedAsync(Driver.INSTANCE :: Nil, Action.async {
```

Comment:
I really don't like the idea of getting ALL routes

Reply:
fixed